src/pages/NewSensor/NewSensorStepThree.js

```diff
@@ -0,0 +1,145 @@
+/* eslint-disable react/forbid-prop-types */
+import React from 'react';
+import { View } from 'react-native';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import { TabContainer } from './TabContainer';
+import {
+  CalendarIcon,
+  HazardIcon,
+  InfoIcon,
+  PaperSection,
+  FlexRow,
+  PageButton,
+  Spacer,
+  WithSpace,
+  TextWithIcon,
+  TextEditor,
+  EditorRow,
+  DateEditor,
+  DurationEditor,
+  TimeEditor,
+} from '../../widgets';
+
+import { WizardActions } from '../../actions/WizardActions';
+import { goBack } from '../../navigation/actions';
+import { selectNewSensor } from '../../selectors/newSensor';
+import { NewSensorActions } from '../../actions/index';
+import { useLoadingIndicator } from '../../hooks/useLoadingIndicator';
+import { DARKER_GREY, LIGHT_GREY, SUSSOL_ORANGE, WHITE } from '../../globalStyles';
+import { buttonStrings, vaccineStrings } from '../../localization';
+import { SECONDS } from '../../utilities/constants';
+
+export const NewSensorStepThreeComponent = ({
+  logInterval,
+  loggingDelay,
+  name,
+  code,
+  updateName,
+  connectToSensor,
+  updateCode,
+  updateLogInterval,
+  updateLoggingDelay,
+  exit,
+  previousTab,
+}) => {
+  const withLoadingIndicator = useLoadingIndicator();
+
+  return (
+    <TabContainer>
+      <WithSpace space={10} horizontal={false} vertical>
+        <PaperSection height={200} headerText={vaccineStrings.new_sensor_step_three_title}>
+          <EditorRow label={vaccineStrings.sensor_name} Icon={<InfoIcon color={DARKER_GREY} />}>
+            <TextEditor size="large" value={name} onChangeText={updateName} />
```

Comment:
is it possible to left align the name text field - currently it's disconnected from the title and is more related to the code area